qiskit/quantum_info/operators/operator.py

```diff
@@ -198,6 +198,99 @@ def from_label(cls, label):
                 op = op.compose(label_mats[char], qargs=[qubit])
         return op
 
+    def apply_permutation(self, perm: list, front: bool = False):
+        """Modifies operator's data by composing it with a permutation.
+
+        Args:
+            perm (list): permutation pattern, describing which qubits
+                occupy the positions 0, 1, 2, etc. after applying the permutation.
+            front (bool): When set to ``True`` the permutation is applied before the
+                operator, when set to ``False`` the permutation is applied after the
+                operator.
+        Returns:
+            Operator: The modified operator.
+
+        Raises:
+            QiskitError: if the size of the permutation pattern does not match the
+                dimensions of the operator.
+        """
+
+        if not front:
```

Comment:
Pretty minor, but since this is an `if/else` block, it's marginally clearer to reason about if the condition is "positive" (i.e. `if front` / `else`).  I get that `not front` is the default, though.

Reply:
changed that, thanks